src/core/font_renderer.js

```diff
@@ -612,17 +637,29 @@ var FontRendererFactory = (function FontRendererFactoryClosure() {
       return fn;
     },
 
-    compileGlyph(code) {
+    compileGlyph(code, glyphId) {
       if (!code || code.length === 0 || code[0] === 14) {
         return noop;
       }
 
+      let fontMatrix = this.fontMatrix;
+      if (this.isCFFCIDFont) {
+        let fdIndex = this.fdSelect.getFDIndex(glyphId);
+        if (fdIndex >= 0 && fdIndex < this.fdArray.length) {
+          let fontDict = this.fdArray[fdIndex];
+          fontMatrix = fontDict.getByName('FontMatrix') ||
+                       [0.001, 0, 0, 0.001, 0, 0];
```

Comment:
Please use `FONT_IDENTITY_MATRIX` from `src/shared/util.js` rather than hard-coding this array here.